bin/install.macosx

```diff
@@ -43,7 +43,7 @@ then
   brew install wget curl
 
   # Install virtualbox.
-  brew cask install virtualbox
+  brew install virtualbox
```

Comment:
this one cannot be executed anymore on big sur (especially on M1)

Reply:
`brew cask` is deprecated, but `brew install` works fine for installing `cask`-ed software on Big Sur (Intel, no idea on M1).